cubedash/_stac.py

```diff
@@ -979,6 +1043,28 @@ def root_catalog():
 # ENDPOINTS
 ##########################
 
+CONFORMANCE_CLASSES = [
+    "http://www.opengis.net/spec/ogcapi-features-1/1.0/conf/core",
+    "http://www.opengis.net/spec/ogcapi-features-1/1.0/conf/geojson",
+    "http://www.opengis.net/spec/ogcapi-features-1/1.0/conf/oas30",
+    "https://api.stacspec.org/v1.0.0-rc.1/core",
```

Comment:
Not for this PR, but hasn't the 1.0.0 been out in the wild for a few years by now, so it should be safe to conform to that API?